src/Makefile

```
@@ -129,6 +135,12 @@ $(foreach file,$(TOINSTALL_NUM_TOP),
 ifeq "$(1)" "legacy"
 $(foreach file,$(TOINSTALL),
 	@echo '  "src/$(file)" {"ocaml/$(file)"}' >> $$@)
+ifneq "$(TOINSTALL_CMXS)" ""
+	@echo 'libexec_root: [' >> $$@
+$(foreach file,$(TOINSTALL_CMXS),
+	@echo '  "src/$(file)" {"ocaml/$(file)"}' >> $$@)
+	@echo ']' >> $$@
+endif
```

This isn't quite right, and was breaking the 4.x build (but CI wasn't displaying that, because of a reasonable - if not useful - opam default!).

```suggestion
	@echo ']' >> $$@
	@echo 'libexec_root: [' >> $$@
$(foreach file,$(TOINSTALL_CMXS),
	@echo '  "src/$(file)" {"ocaml/$(file)"}' >> $$@)
```

opam doesn't mind empty sections in .install files, so on a static system having:

```
libexec_root: [
]
```
is fine.